package.json

```diff
@@ -14,6 +14,7 @@
     "node-sass": "^6.0.1",
     "react": "^17.0.2",
     "react-dom": "^17.0.2",
+    "react-icons": "^4.3.1",
```

Comment:
remove ^ in general so we are locked into a specific version, and upgrade of version is then a decision rather than a chance

Reply:
Alright, thanks Arnab - first comment of my commit :)